.circleci/config.yml

```diff
@@ -21,7 +21,6 @@ commands:
       - setup-build-workspace
       - setup-shared-env:
           study_key: << parameters.study_key >>
-      - set-deployment-environment
       - build-app
```

Comment:
When we build and store (like when we create a hotfix_* or rc_* branch), we might not have a branch that we can map to an environment, so this will fail.